apps/nextjs-website/src/lib/strapi/fetches/fetchGuides.ts

```diff
@@ -1,20 +1,21 @@
 import * as qs from 'qs';
 import { fetchFromStrapi } from '@/lib/strapi/fetchFromStrapi';
 import { GuidesCodec } from '@/lib/strapi/codecs/GuidesCodec';
+import { productPopulate } from './fetchProducts';
 
 const makeStrapiGuidesPopulate = () =>
   qs.stringify({
-    populate: [
-      'image',
-      'mobileImage',
-      'listItems',
-      'versions',
-      'bannerLinks.icon',
-      'seo',
-      'seo.metaSocial.image',
-      'product.logo',
-      'product.bannerLinks.icon',
-    ],
+    populate: {
+      image: { populate: '*' },
```

Comment:
Why do we need the nested `populate`?

Reply:
Because i need to use here the new helper for product `productPopulate` and `qs` does not support both inline and nested populate so i had to convert every field in this way